public/emoji/flags.css

```diff
@@ -0,0 +1,2069 @@
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e7-1f1e6 {
+            background-position: -1px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1ff {
+            background-position: -35px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1fd {
+            background-position: -69px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1fc {
+            background-position: -103px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1fa {
+            background-position: -137px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1f9 {
+            background-position: -171px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1f8 {
+            background-position: -205px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1f7 {
+            background-position: -239px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1f6 {
+            background-position: -273px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1f4 {
+            background-position: -307px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1f2 {
+            background-position: -341px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1f1 {
+            background-position: -375px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1ee {
+            background-position: -409px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1ec {
+            background-position: -443px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1eb {
+            background-position: -477px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1ea {
+            background-position: -511px -511px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-flags-1f1e6-1f1e9 {
+            background-position: -545px -1px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        @media screen and (-webkit-min-device-pixel-ratio: 1.0), screen and (min--moz-device-pixel-ratio: 1.0), screen and (-o-min-device-pixel-ratio: 100/100), screen and (min-device-pixel-ratio: 1.0), screen and (min-resolution: 1.0dppx) {
+            .em-flags-1f6a9,
```

Comment:
^

Reply:
This is a generated file :slightly_smiling_face: